### PR TITLE
feat(tutorial): complete step 5

### PR DESCRIPTION
**Pull request by pulakb**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Comment by pulakb** [comment]
@kellychurchill the url is [https://carbon-tutorial-pulakb.eu-gb.mybluemix.net/#/](https://carbon-tutorial-pulakb.eu-gb.mybluemix.net/#/)

File: src/index.scss [hunk 1]
```diff
@@ -1,29 +1,16 @@
-$feature-flags: (
-  ui-shell: true,
-  grid-columns-16: true,
-);
```
Comment (kellychurchill) [inline src/index.scss]: These should not be removed - it is causing additional padding in your layout.

Comment (pulakb) [reply src/index.scss]: @kellychurchill i have updated it. Please check.

**Comment by kellychurchill** [comment]
Your layout still has extra padding. 

Your `index.scss` file should look like this: https://github.com/kellychurchill/carbon-tutorial/blob/88069ee2c515c72b2d662f1b9f847d33b08eaaa2/src/index.scss

Please review your preview link before requesting review: https://deploy-preview-1464--carbon-tutorial.netlify.com

**Comment by pulakb** [comment]
> Your layout still has extra padding.
> 
> Your `index.scss` file should look like this: https://github.com/kellychurchill/carbon-tutorial/blob/88069ee2c515c72b2d662f1b9f847d33b08eaaa2/src/index.scss
> 
> Please review your preview link before requesting review: https://deploy-preview-1464--carbon-tutorial.netlify.com

@kellychurchill https://github.com/kellychurchill/carbon-tutorial/blob/88069ee2c515c72b2d662f1b9f847d33b08eaaa2/src/index.scss has some additional code which i could not find in the tutorial.... Anyway i will update the PR

**Comment by kellychurchill** [comment]
I will approve. Technically, your IBM Cloud app should also reflect the updates, only your Netlify preview shows the updates.

**Comment by pulakb** [comment]
Thanks @kellychurchill. Can you please close this PR?

**Pull request closed without merging**